### ISSUE TITLE
Compile fails on Windows

### ISSUE THREAD

**Issue by kraxarn** [whoshuu/cpr#139]
Getting the following error:

```
-- Performing Test curl_cv_recv
-- Performing Test curl_cv_recv - Failed
CMake Error at opt/cpr/opt/curl/CMake/OtherTests.cmake:86 (message):
  Unable to link function recv
Call Stack (most recent call first):
  opt/cpr/opt/curl/CMakeLists.txt:1005 (include)
```

**Comment by whoshuu** [comment]
@KraXarN thanks for bringing up this issue. It looks like a curl specific problem. When searching for `Unable to link function recv` most of the results I get are related to curl + CMake. Perhaps that'd be a good starting point for investigating the issue further.

I hesitate to say that the issue can be resolved in this library alone, except for two cases. A) an upgrade of the embedded curl library resolves the issue trivially. Or B) there's an option in the curl CMake build scripts that I can expose on this project that would resolve the issue.

**Comment by KraXarN** [comment]
Yea, it's probably more of a curl issue. Trying to compile curl just by itself results in the same error. Probably a bug with curl, although I'm not very used to compiling stuff on Windows...

**Comment by ilkeryasinaksoy** [comment]
@KraXarN look at this: [](https://github.com/whoshuu/cpr/issues/126#issuecomment-320880003)

**Comment by COM8** [comment]
Should be fixed in #126.
Feel free to open a new issue in case this is still a problem.
